docs/source/estimation.rst

```diff
@@ -35,31 +35,28 @@ argument of any criterion function optimized with estimagic's :ref:`minimize`.
 
 It can have the following columns (most of them being optional)
 
-- 'value' (dtype=float): Start values for the optimization. If not provided,
-  start values based on constraints and other information in params are drawn
-  randomly. If provided, we ensure that these values    are taken as start
-  values for local optimizers and are in the initial population of genetic
-  algorithms or pseudo global optimizers.
-- 'lower' (dtype=float): Lower bounds enforced in optimization. Can take the
+- `'value'`` (dtype=float): Start values for the optimization. This column is
+  not optional. These values are taken as start values for local optimizers
+  and are in the initial population of genetic algorithms or pseudo global
+  optimizers.
+- `'lower'`` (dtype=float): Lower bounds enforced in optimization. Can take the
   values - np.inf or np.nan for unbounded parameters.
-- 'upper' (dtype=float): Upper bounds enforced in optimization. Can take the
+- `'upper'`` (dtype=float): Upper bounds enforced in optimization. Can take the
   values np.inf or np.nan for unbounded parameters.
-- 'draw_lower' and 'draw_upper' (dtype=float): bounds used to randomly draw
+- `'draw_lower'`` and `'draw_upper'`` (dtype=float): bounds used to randomly draw
   parameters. They are ignored or only enforced approximately for some
   constrained parameters. These columns are required for genetic or pseudo
-  global optimizers or if 'value' is not provided.
-- 'fixed' (dtype=bool): Indicates if a parameter is fixed to value. If any
-  parameter is fixed, 'value' is required.
-- 'group' (dtype=str or None): Indicates in which group (if any)
+  global optimizers.
+- `'group'`` (dtype=str or None): Indicates in which group (if any)
   a parameter's values will be plotted in the convergence tab of the dashboard.
   Parameters with value None are not plotted.
 
 It is important to distinguish three related but different concepts:
 
-- params_df: the DataFrame described above
-- params_sr: the 'value' column of params. This is the first argument of any
+- `params_df`: the DataFrame described above, sometimes just called `params`.
```

Comment:
And I think you need to ticks for each side to make it literal. Otherwise it is italic or bold.

Reply:
Thanks, fixed.